pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
> We don't need to distinguish between images... ?

Wouldn't you be able to recognize builder pods based on build related labels?  There was the discussion of forcing certain labels onto resources during admission last week, so you could write those rules in a different location.

Doesn't it seem likely that a paranoid admin would say "match all" and we'd recommend "match build labels" for usability?